Sources/Apodini/Semantic Model Builder/Traversable.swift

```diff
@@ -194,15 +189,15 @@ private func execute<Element, Target>(
 
         switch child {
         case let target as Target:
-            assert(((try? typeInfo(of: property.type).kind) ?? .none) == .struct, "\(Target.self) \(property.name) on element \(info.name) must be a struct")
+            assert(((try? typeInfo(of: property.type).kind) ?? .none) != .class, "\(Target.self) \(property.name) on element \(info.name) must be a struct")
```

Comment:
Seems like the assert messages doesn't really reflect the assert condition anymore(?) [applies to all asserts in this file?].
Also, to my knowledge, asserts are not present in the final release build, are they(?). Might be sensible to switch to [precondition](https://developer.apple.com/documentation/swift/1540960-precondition)?

Reply:
I forgot to update the message. Should be fixed now.

Yes, asserts are not executed in the release build and that is on purpose. The conditions contain extra `typeInfo` calls, which are quite expensive. Thus I thought it would not be worth it. I mean if someone really manages to introduce a class type Handler to a release build without ever starting the application in dev mode, it is their fault I guess🤷‍♂️